pkg/config/types/orchestrator.go

```diff
@@ -17,6 +17,8 @@ type Orchestrator struct {
 	NodeManager      NodeManager      `yaml:"NodeManager,omitempty" json:"NodeManager,omitempty"`
 	Scheduler        Scheduler        `yaml:"Scheduler,omitempty" json:"Scheduler,omitempty"`
 	EvaluationBroker EvaluationBroker `yaml:"EvaluationBroker,omitempty" json:"EvaluationBroker,omitempty"`
+	// SupportReverseProxy configures the orchestrator node to run behind a reverse proxy
+	SupportReverseProxy bool `yaml:"SupportReverseProxy,omitempty" json:"SupportReverseProxy,omitempty"`
```

Comment:
nit: Suggest renaming to `ReverseProxy` - it's more concise while maintaining clarity since the boolean field already implies feature support.

Reply:
You are totally right. Go for `SupportReverseProxy` as you suggested